pkg/oc/admin/migrate/migrate.go

```diff
@@ -28,3 +31,15 @@ func NewCommandMigrate(name, fullName string, f *clientcmd.Factory, out, errOut
 	cmd.AddCommand(cmds...)
 	return cmd
 }
+
+func NewCommandMigrateAlpha(name, fullName string, f *clientcmd.Factory, out, errOut io.Writer, cmds ...*cobra.Command) *cobra.Command {
```

Comment:
I feel like we are starting to abuse the migrations we do via ansible.

Reply:
Why?

Reply:
This wouldn't be for ansible.  This is a convenience for someone who wants to move alpha annotations forward without having to write the script themselves.